### PR TITLE
Fix for incorrect 'tester' class

### LINKED ISSUE #13: microbenchmark results reported as missing despite proper results in LAVA
Example: https://art-reports.linaro.org/#/build/451
Likely JSON format changed. Needs urgent fix.

### PR DESCRIPTION
**Pull request by mwasilew**
This commit fixes #13. In case the test job is complete before task is
created, incorrect class is used to fetch results. In addition this
patch adds feature to force result fetching on the test jobs that have
proper results but art-reports fails to retrieve them. Fetching results
doesn't require re-submitting test jobs.

Signed-off-by: Milosz Wasilewski <milosz.wasilewski@linaro.org>

### CODE REVIEW

File: benchmarks/admin.py [hunk 1]
```diff
@@ -18,7 +18,12 @@ class ResultAdmin(admin.ModelAdmin):
 @admin.register(TestJob)
 class TestJobAdmin(admin.ModelAdmin):
     list_display = ('id', 'url', 'status', 'completed', 'created_at')
+    actions = ['force_fetch_results']
```
Comment (terceiro) [inline benchmarks/admin.py]: this does not seem to be used

File: benchmarks/admin.py [hunk 2]
```diff
@@ -18,7 +18,12 @@ class ResultAdmin(admin.ModelAdmin):
...
 
+    def force_fetch_results(self, request, queryset):
+        for testjob in queryset.all():
+            set_testjob_results.delay(testjob)
+    force_fetch_results.short_description = "Force fetch results"
```
Comment (terceiro) [inline benchmarks/admin.py]: nor this?

Comment (terceiro) [inline benchmarks/admin.py]: oops, please ignore this. I think I'm not seeing the whole picture here

Comment (mwasilew) [inline benchmarks/admin.py]: Probably my fault. This is only visible in admin interface. It might be a separate patch so it doesn't 'pollute' the fix for the bug. The reason for including this code is to provide ability to 're-fetch' results that art-reports is missing. So in a way it also 'fixes' the bug (or rather the results of the bug).

File: benchmarks/admin.py (hunk 1 above)
Comment (mwasilew) [inline benchmarks/admin.py]: only in admin interface

File: benchmarks/admin.py (hunk 2 above)
Comment (terceiro) [inline benchmarks/admin.py]: nah, LGTM
